src/main/resources/assets/appliedenergistics2/lang/de_de.lang

```diff
@@ -358,12 +384,38 @@ waila.appliedenergistics2.Unlocked=Entsperrt
 waila.appliedenergistics2.Showing=Zeigt
 waila.appliedenergistics2.Contains=Enthält
 waila.appliedenergistics2.Channels=%1$d von %2$d Kanälen
+waila.appliedenergistics2.P2PUnlinked=Link getrennt
```

Comment:
"Unlinked" means that the P2P device is not linked, in German that would be "Nicht verlinkt" and not "Link disconnected".